docker/iqdb/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ruby:2.7.3-alpine
+
+ADD https://api.github.com/repos/zwagoth/iqdbs/git/refs/heads/master /tmp/iqdbs_version.json
+COPY iqdb.patch /tmp/iqdb.patch
+ENV IQDB_VERSION=20161008
+
+RUN apk --no-cache add build-base git libjpeg-turbo-dev gd-dev bash \
+  # iqdbs
+  && cd /tmp \
+  && wget -q  https://iqdb.org/code/iqdb-$IQDB_VERSION.tar.bz2 \
```

Comment:
You should probably compare checksums of the file to ensure it won't change from underneath you